docs/examples/advanced/imagenet/README.rst

```diff
@@ -156,7 +156,11 @@ we do not include it here, but you can find it in the GitHub repository `here
    # They can either be set here or as early as possible in the Python script.
    # Use `uv run --offline` on clusters without internet access on compute nodes.
    # Using `srun` executes the command once per task, once per GPU in our case.
-   srun bash -c \
+   # --gres-flags=allow-task-sharing is required to allow multiple tasks to share
+   # the same GPU and avoid a mysterious NCCL error in
```

Comment:
The wording could perhaps be slightly different, even if you might be technically correct(?). It's so that tasks on the same node can see the GPUs of other tasks, even with `--gpus-per-task=1`, which would otherwise have each task only see its own GPU. This is something to do with CGroups:
- https://mila-umontreal.slack.com/archives/C08KMD8FJNA/p1762016455249749?thread_ts=1759420220.288619&cid=C08KMD8FJNA
- https://support.schedmd.com/show_bug.cgi?id=17875

```suggestion
   # --gres-flags=allow-task-sharing is required to allow multiple tasks to share
   # the same GPU and avoid a mysterious NCCL error in
```